src/constant/enums.js

```diff
@@ -371,4 +371,9 @@ export function getIdBasedOnLabel(enumList, label) {
         console.warn('Label should be a string');
         return null;
     }
-}
\ No newline at end of file
+}
+export const PriorityLables = Object.freeze({
```

Comment:
We use `..Labels` for the string mapping.
Update the enum name to `RequestPriority`